readthedocs/vcs_support/backends/bzr.py

```diff
@@ -17,7 +17,7 @@ class Backend(BaseVCS):
     supports_tags = True
     fallback_branch = ''
 
-    def update(self):
+    def update(self, version=None):  # pylint: disable=arguments-differ
```

Comment:
This should be added in the parent class to avoid the linter error, but I'm not sure this is needed

Reply:
Also, we have https://github.com/rtfd/readthedocs.org/blob/7c6abc78635932852b48f0281551f9a7616f2acd/readthedocs/vcs_support/backends/git.py#L30-L30, but looks like we aren't using, it should set in the parent class too

Reply:
Oh! I used it in the initial implementation, forgot to remove it, I'll remove it. Thanks :)

Reply:
I think a better approach is pass this data to https://github.com/rtfd/readthedocs.org/blob/7c6abc78635932852b48f0281551f9a7616f2acd/readthedocs/projects/models.py#L811-L811 (version can be an object) and put what we need in https://github.com/rtfd/readthedocs.org/blob/7c6abc78635932852b48f0281551f9a7616f2acd/readthedocs/vcs_support/base.py#L53-L58

Reply:
Okay I'll try this out

Reply:
@stsewd Updated #5838 please have a look https://github.com/rtfd/readthedocs.org/pull/5838/commits/cdd38a1537210fe89191769eb1a74de1266d6179

Reply:
That's better, we only need to pass the version object instead of the slug, because we don't have access to the db from there, version is a wrapper from the api response, but it has the same fields as the model, except it can't be used to make queries